documentation/modules/creating-migration-plan-2-8.adoc

```diff
@@ -41,84 +41,232 @@ ifdef::vmware[]
+** To preserve static IPs, do the following:
+
+*** Click the *Edit* icon.
+*** Toggle the *Whether to preserve the static IPs* switch.
+*** Click *Save*.
 +
 {project-short} then issues a warning message about any VMs whose vNIC properties are missing. To retrieve any missing vNIC properties, run those VMs in vSphere. This causes the vNIC properties to be reported to {project-short}.
 
 * *Disk decryption passphrases*: For disks encrypted using Linux Unified Key Setup (LUKS). 
 
-** To enter a list of decryption passphrases for LUKS-encrypted devices, in the *Settings* section, click the *Edit* icon next to *Disk decryption passphrases*, enter the passphrases, and then click *Save*. You do not need to enter the passphrases in a specific order. For each LUKS-encrypted device, {project-short} tries each passphrase until one unlocks the device.
+** To enter a list of decryption passphrases for LUKS-encrypted devices, do the following:
+
+*** Click the *Edit* icon.
+*** Enter the passphrases.
+*** Click *Save*.
++
+You do not need to enter the passphrases in a specific order. For each LUKS-encrypted device, {project-short} tries each passphrase until one unlocks the device.
 
 * *Root device*: Applies to multi-boot VM migrations only. By default, {project-short} uses the first bootable device detected as the root device.
 
-** To specify a different root device, in the *Settings* section, click the *Edit* icon next to *Root device* and then either select a device from the list or enter a device in the text box.
+** To specify a different root device, do the following:
+
+*** Click the *Edit* icon next to *Root device*.
+*** Either select a device from the list or enter a device in the text box.
+*** Click *Save*.
 +
 {project-short} uses the following format for disk location: `/dev/sd<disk_identifier><disk_partition>`. For example, if the second disk is the root device and the operating system is on the disk's second partition, the format would be: `/dev/sdb2`. After you enter the boot device, click *Save*.
 +
 If the conversion fails because the boot device provided is incorrect, it is possible to get the correct information by checking the conversion pod logs.
 
-endif::[]
+* *Shared disks*: Applies to cold migrations only. Shared disks are disks that are attached to multiple VMs and that use the multi-writer option. These characteristics make shared disks difficult to migrate. By default, {project-short} does not migrate shared disks.
++
+[NOTE]
+====
+Migrating shared disks might slow down the migration process.
+====
++
+** To migrate shared disks in the migration plan, do the following:
+
+*** Click the *Edit* icon.
+*** Toggle the *Migrate shared disks* switch.
+*** Click *Save*.
+
+* *PVC name template*: Generates a persistent volume claim (PVC) name for a VM disk that overrides the template set in the plan.
```

Comment:
"....for a VM disk that overrides the template set in the plan."

IIUC, the VM specific template that will override the template set in the plan is set via the VMs tab (not documented yet and not mentioned here).
So for me the above description is a bit confusing. 

This is relevant for other 3 template fields as well.

Reply:
@sgratch Fixed; the VM tab was included before, but the fact that setting it overrides the Plan Details page wasn't. 

Please review the PR again, thanks.